tardis/montecarlo/montecarlo_numba/single_packet_loop.py

```diff
@@ -55,6 +79,15 @@ def single_packet_loop(
         and if virtual packets are requested - also updates the vpacket_collection
     """
 
+    with objmode():
+        # if r_packet.index in montecarlo_configuration.list_of_index:
+        if (
+            montecarlo_configuration.seed_index
+            and r_packet.seed == montecarlo_configuration.seed_index
+        ):
```

Comment:
Could the seed test be done in the packet_tracking function?

Reply:
Yes I have changed the implementation 😄